frontend/app/fragments/public-schools/FutureResidentialDevelopment.js

```diff
@@ -1,4 +1,4 @@
-import { computed } from EmberObject from '@ember/object';
```

Comment:
Yeah, this doesn't look right, weird that autofix did it this way. I don't think it's even valid syntax.

Reply:
@allthesignals it's not and threw a build error.  Was super weird that `eslint` with the `--fix` flag wrote it this way